studia-historica-upsaliensa.csl

```diff
@@ -0,0 +1,588 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" and="symbol" initialize="false" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Studia Historica Upsaliensa </title>
+    <title-short>Studia</title-short>
+    <id>http://www.zotero.org/styles/studia-historica-upsaliensa</id>
+    <link href="http://www.zotero.org/styles/studia-historica-upsaliensa" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
+    <link href="http://www.hist.uu.se" rel="documentation"/>
```

Comment:
I thought about that. Yes there is [http://hist.uu.se/Student/Masterutbildning/Skrivauppsats/Formaliainstruktioner/tabid/5600/language/en-US/Default.aspx](url)
The problem is that the information on that pages changes URL every time we update our homepage, something we do every three to four years. A few years from now the link above will most certainly be dead, but the information will still be found somewhere else on hist.uu.se